Dockerfile

```diff
@@ -6,11 +6,13 @@ FROM golang:1.15 as openelb-builder
 
 COPY / /go/src/github.com/openelb/openelb
 
+RUN go env -w GOPROXY=https://goproxy.cn,direct
+
 WORKDIR /go/src/github.com/openelb/openelb
-RUN GO111MODULE=on CGO_ENABLED=0 go install -i -ldflags '-w -s' github.com/openelb/openelb/cmd/...
-RUN GO111MODULE=on CGO_ENABLED=0 go install -i -ldflags '-w -s' github.com/osrg/gobgp/cmd/gobgp
+RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GOARM=6 go install -i -ldflags '-w -s' github.com/openelb/openelb/cmd/...
+RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GOARM=6 go install -i -ldflags '-w -s' github.com/osrg/gobgp/cmd/gobgp
```

Comment:
I remember that GOARM is only effective when GOARCH is arm64.

Reply:
Can you tell me why you only set GOARCH to amd64?

Reply:
In fact, we are no longer using this DockerFile.